packages/components/inputs/localized-money-input/src/localized-money-input.js

```diff
@@ -157,83 +143,66 @@ const LocalizedMoneyInput = (props) => {
+                hasError={Boolean(
+                  props.hasError || (props.errors && props.errors[currency])
+                )}
+                hasWarning={Boolean(
+                  props.hasWarning ||
+                    (props.warnings && props.warnings[currency])
+                )}
+                intl={intl}
+                warning={props.warnings && props.warnings[currency]}
+                error={props.errors && props.errors[currency]}
+                {...createLocalizedDataAttributes(props, currency)}
+              />
             );
-          return (
-            <LocalizedInput
-              key={currency}
-              id={LocalizedMoneyInput.getId(id, currency)}
-              name={LocalizedMoneyInput.getName(props.name, currency)}
-              value={props.value[currency]}
-              onChange={props.onChange}
-              currency={currency}
-              placeholder={
-                props.placeholder ? props.placeholder[currency] : undefined
-              }
-              onBlur={props.onBlur}
-              onFocus={props.onFocus}
-              isDisabled={props.isDisabled}
-              isReadOnly={props.isReadOnly}
-              currenciesControl={(() => {
-                if (
-                  !hasRemainingCurrencies ||
-                  props.hideCurrencyExpansionControls
-                )
-                  return null;
-                if (isFirstCurrency && !areCurrenciesExpanded)
-                  return (
-                    <CurrencyControl
-                      isClosed={true}
-                      onClick={toggleCurrencies}
-                      remainingCurrencies={currencies.length - 1}
-                    />
-                  );
-                if (isLastCurrency)
-                  return (
-                    <CurrencyControl
-                      onClick={toggleCurrencies}
-                      remainingCurrencies={currencies.length - 1}
-                      isDisabled={Boolean(
-                        props.hasError ||
-                          hasErrorOnRemainingCurrencies ||
-                          props.hasWarning ||
-                          hasWarningOnRemainingCurrencies
-                      )}
-                    />
-                  );
-                return null;
-              })()}
-              hasError={Boolean(
-                props.hasError || (props.errors && props.errors[currency])
-              )}
-              hasWarning={Boolean(
-                props.hasWarning || (props.warnings && props.warnings[currency])
-              )}
-              intl={intl}
-              warning={props.warnings && props.warnings[currency]}
-              error={props.errors && props.errors[currency]}
-              {...createLocalizedDataAttributes(props, currency)}
-            />
-          );
-        })}
+          })}
+        </Stack>
+        {shouldRenderCurrencyControl && (
+          <LocalizedInputToggle
```

Comment:
This is the "currenciesControl" now. It is always rendered in the same place, instead of rendering conditionally with one of the LocalizedInput components